### PR TITLE
Add gstlal

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This adds a recipe for GstLAL.

cc: @myNameIsPatrick as co-maintainer, can you take a look through and make sure I haven't botched anything?

### CODE REVIEW

File: recipes/gstlal/build.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# conda-forge/conda-forge.github.io#621
+find ${PREFIX} -name "*.la" -delete
+
```
Comment (myNameIsPatrick) [inline recipes/gstlal/build.sh]: This is missing `./00init.sh` before running `./configure`.

**Comment by myNameIsPatrick** [comment]
@duncanmmacleod, I think this looks good for the most part. I added a comment in `build.sh` with a missing step.

File: recipes/gstlal/build.sh (hunk 1 above)
Comment (duncanmmacleod) [reply recipes/gstlal/build.sh]: @myNameIsPatrick, I don't think the `00init.sh` is required when you build from the tarballs.

Comment (myNameIsPatrick) [reply recipes/gstlal/build.sh]: @duncanmmacleod, my bad, I didn't realize this. Ignore my previous comment.

**Comment by duncanmmacleod** [comment]
@conda-forge/help-python-c, I think this recipe is ready for review, thanks.

**Pull request merged**